examples/java-protovalidate/proto/example/v1/user.proto

```
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package example.v1;
```

Buf (1.54.0)

Files with package "example.v1" must be within a directory "example/v1" relative to root but were in directory "examples/java-protovalidate/proto/example/v1".

(PACKAGE_DIRECTORY_MATCH)